### ISSUE TITLE
Wrong redirect on auth

### ISSUE THREAD

**Issue by diver-ua** [babelouest/glewlwyd#69]
the callback ur is `https://172.18.0.2/Oid/authform.htm`
The 302 redirect  to that url looks like  `Location: https://172.18.0.2/Oid/authform.html#state=%26state%3deb9b84b8b40941878101565e75da6f08&access_token=eyJhbG`

the state param is incorrect...

**Comment by babelouest** [comment]
Hello @diver-ua ,

I'm gonna need more information for this one, all my tests with the state pass but I probably missed one. So to fix it I need to reproduce it.

Can you post the full urls your browser follows from the client page to the callback url?
Hint: press `<F12>` when your browser is in its first state in the client app, check the `keep the logs`.

You can anonymize the privacy data in the urls like the client_id or the token value value if you need to.

I need the urls like
- step 1: `https://172.18.0.2/Oid/authform.htm`
- step 2: `https://glewlwyd.address/api/oidc/auth?client_id=xxx&scope=scope1+scope2&response_type=token&[...]`
- step 3: `https://172.18.0.2/Oid/authform.htm#state=xyz&access_token=abc[...]`

**Comment by diver-ua** [comment]
```
GET https://172.18.0.2/Oid/
GET https://172.18.0.2/Oid/ru_RU/
GET https://auth.itstep.kiev.ua/api/oidc/auth?client_id=1c1234567890QWERTYUIOP&redirect_uri=https%3a%2f%2f172.18.0.2%2fOid%2fauthform.html&response_type=id_token+token&scope=openid+g_profile&state=811ad5cbc0c54e8692b6e1fa070737ae&nonce=c6f54ac3827e4cfbbc552c96d4456134&g_continue
GET https://172.18.0.2/Oid/authform.html#state=%26state%3d811ad5cbc0c54e8692b6e1fa070737ae&access_token=eyJhbGciOiJSUzI1NiIsImtpZCI6IkVmNjRaOUhrQWNHZFRha0R4blR1ak5qQmc3Q3Vua2FwMGIxZkxBV182eDQiLCJ0eXAiOiJKV1QifQ.eyJhdWQiOiIxYzEyMzQ1Njc4OTBRV0VSVFlVSU9QIiwiZXhwIjoxNTc1MjkxMTcxLCJpYXQiOjE1NzUyODc1NzEsImlzcyI6Imh0dHBzOi8vYXV0aC5pdHN0ZXAua2lldi51YSIsInNhbHQiOiJwQU95UGt6TEFLdjkydHJIIiwic2NvcGUiOiJvcGVuaWQgZ19wcm9maWxlIiwic3ViIjoiNW4xVHg0azZYVzY5a0ozbUphQlF6T05CanVFenhmMGgiLCJ0eXBlIjoiYWNjZXNzX3Rva2VuIn0.e5iqKB3WG5d7q6y_7QRQpS4_4U8vdeNsn7P5zty0cib1NVKlG-DrKuGFgFiMDZ_THtMxj8N8EkA9prNm6g0XmYNz0K6X9NlmINr6PbQC_pEb1wXelwVLV0ranPOC2QlXYnbtuzQbd6P-5_NLhccxP3gscOIGmJFXMkfGAKkPy0a5ZHw25Nb1LS4QzLdh6-Mg97g-lxxAWyvbk0j4goq_ZVq7hMg-cxLGjV-zqYOCRUQNzGLVNSMJFfZpL_BE-m4VM3Vomnm6wVlwd6QOC8yMxMHFH7fskSv4iGvgAQrlWjfs1AmyXlX02qw_3qgnsK9rzqNmdhGUiP1wmBlN4LwB-g&token_type=bearer&expires_in=3600&iat=1575287571&scope=openid+g_profile&id_token=eyJhbGciOiJSUzI1NiIsImtpZCI6IkVmNjRaOUhrQWNHZFRha0R4blR1ak5qQmc3Q3Vua2FwMGIxZkxBV182eDQiLCJ0eXAiOiJKV1QifQ.eyJhbXIiOlsicGFzc3dvcmQiXSwiYXRfaGFzaCI6InBoM2V5bllwWXlNdUk1eTBtNURIbWciLCJhdWQiOiIxYzEyMzQ1Njc4OTBRV0VSVFlVSU9QIiwiYXV0aF90aW1lIjoxNTc1Mjg3NDMwLCJhenAiOiIxYzEyMzQ1Njc4OTBRV0VSVFlVSU9QIiwiZW1haWwiOiJzdHVkZW50QGl0c3RlcC5raWV2LnVhIiwiZXhwIjoxNTc1MjkxMTcxLCJpYXQiOjE1NzUyODc1NzEsImlzcyI6Imh0dHBzOi8vYXV0aC5pdHN0ZXAua2lldi51YSIsIm5hbWUiOiJUaGUgQWRtaW5pc3RyYXRvciIsIm5vbmNlIjoiYzZmNTRhYzM4MjdlNGNmYmJjNTUyYzk2ZDQ0NTYxMzQiLCJzdWIiOiI1bjFUeDRrNlhXNjlrSjNtSmFCUXpPTkJqdUV6eGYwaCJ9.f6KlfMKEp1bOrpWvckR3fmvOAqdmolKoFo3COZ3xF5Cjy20tCdR3lAN21sqs6ul4O25S1pf2syHwDZK_cW_O6b2BREfIqEeX6VZ_wytXYBS5yXn19BR-tnvSHTsJyFnHUFjQd2TlChxLB45Q_fHYdT6VKvd_56_k6byUg7FKND2j30DSU5bg8QSpSZ7tJPBJ77scSUAWZLPuyv5126Wd6NwP7glYHHPcaKATpEDuZN0w62KX762uTBAeM062eV-8YMsxSvg7VBnLR7FA2TGWJnylDf2lGMnx10IOymyTIBDoxeJQRxkopIdBrDev0JQH_xnX8unfWmhMxGEmXb8jLA

```

**Comment by babelouest** [comment]
Hello @diver-ua ,

This should be fixed in bf2c631 now, thanks for reporting the issue!
